test/trivial/diten/lcs/lcs.chpl

```diff
@@ -20,12 +20,12 @@ proc main() {
   len1 = (s1.size);
```

Comment:
I'd prefer to revert the changes to this file, then make changes to just two lines. Line 52 becomes:
`if s1[i-1] == s2[j-1] {`

And line 77 becomes:
`write(X[i-1]);`

Reply:
OK, thanks, done in https://github.com/chapel-lang/chapel/pull/14671/commits/d673888acb65ddd2934464b152f6bdb9b155ae76